app/models/search/tag_query.rb

```diff
@@ -109,7 +109,13 @@ def suggested_character_filter
   end
 
   def to_wrangle_filter
-    term_filter(:to_wrangle, bool_value(options[:to_wrangle])) unless options[:to_wrangle].nil?
+    return [] unless options[:to_wrangle]
+
+    [
+      { bool: { should: [{ range: { uses: { gt: 0 } } }, term_filter(:canonical, true)] } }, # Check if used OR canonical
+      term_filter(:unwrangleable, false),
+      term_filter(:unwrangled, true)
+    ]
```

Comment:
Wouldn't this still need a `unless options[:to_wrangle].nil?` at the end?

Reply:
Isn't this case covered in the early return just above?

I note that past me didn't covered the case where someone wants to explicitly request tags which are _not_ to wrangle (the filter is similarly skipped on `nil` or `false`), probably both because there is no use case for it as of now and the request sounded complicated. Should I add it nonetheless to avoid some future contributor being surprised by the behavior? Maybe out of scope and just leave an explicit comment instead?

Reply:
Ah yes, I was looking at a version of the diff that hid that return. I'd say we should add the `.nil?` since it's not a large change and avoids the confusion with Ruby's falsiness